src/main/java/Deadline.java

```diff
@@ -0,0 +1,26 @@
+import javax.sound.midi.SysexMessage;
+
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by){
```

Comment:
Though it doesn't affect your runtime execution, it is preferred to maintain layout consistency in terms of keeping a space before the  `{` here.